### ISSUE TITLE
[FEATURE] Make the player act more like a native player (mostly android focused)

### ISSUE THREAD

**Issue by NatanielRegula** [jhomlala/betterplayer#557]
**Is your feature request related to a problem? Please describe.**
1. There is no way (to the best of my knowledge) to pause and start the video using buttons on your headset. For example clicking play on Bluetooth headphones does nothing. 

2. The player doesn't request audio focus meaning that it doesn't stop other audio sources on the device as well as isn't stopped when other audio interrupts it.

**Describe the solution you'd like**
1. The player should start or stop playback when user clicks on media buttons on his/her headphones.
2. It should request audio focus and handle interruptions.

**Describe alternatives you've considered**
I think the package audio_service has the ability to perform these actions but I didn't managed to get it working.

**Comment by jhomlala** [comment]
@NatanielRegula I agree this may be good addition, but there're more required features of player. I'll leave this ticket open, maybe you or someone else will have time to implement this feature. Right now I don't see chance to implement this on my side in near future.
